docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/arithmetic/segment-max-16.rst

```diff
@@ -0,0 +1,102 @@
+SegmentMax
+===================
```

Comment:
```suggestion
SegmentMax
==========
```
Should number `=` be same as `heading` length?

Reply:
Fixed.